### ISSUE TITLE
Display which values are not consistent with the scheme

### ISSUE THREAD

**Issue by hesamsagha** [audeering/audformat#303]
Description:

When a value is not consistent with the given scheme, it would be hard to debug to find which values cause it. 
The audformat just displays `ValueError: Some value(s) do not match scheme`.

Feature request:
Add a report showing which values are not in the scheme. Something like this code-snippet in the column.py

``` 
  except ValueError:
      illegal = [value not in scheme for value in values]
      raise ValueError(values[illegal])
```

**Comment by hagenw** [comment]
This would indeed help finding problematic entries.

One problem is that the number of values can be quite huge (>10000), so I would suggest we only show the first 50 entries and add `, ...` if there are more. Would this be ok with you?

**Comment by hesamsagha** [comment]
yes, even first 10 would be good. You may have `unique(values[illegal])` to make it shorter

**Comment by hesamsagha** [comment]
let's keep the first 50 :)
